app/src/main/java/com/worldsnas/starplayer/model/LocalMusicProviderImpl.kt

```diff
@@ -21,12 +20,12 @@ constructor(private val contentResolver: ContentResolver) : LocalMusicProvider {
     )
     private val uri: Uri = MediaStore.Audio.Media.EXTERNAL_CONTENT_URI
 
-    override suspend fun getAllMusic(): List<Music> {
+    override suspend fun getAllMusic(): List<LocalMusic> {
```

Comment:
hamchenin

Reply:
na dige, ta Repo Local Music biaram baad tabdil konm be Music

Reply:
`LocalMusic` vazifash chize dige ie
in vazayef ro ghati nakonim zendegi shirin khahad shod!